src/models/recipe.ts

```diff
@@ -10,5 +10,56 @@ class Recipe {
     this.image = recipeImage;
   }
 }
+interface MissedIngredients {
+  amount: number;
+  name: string;
+  unit: string;
+  unitLong: string;
+  image: string;
+  id: number;
+}
+
+interface UsedIngredients {
+  amount: number;
+  name: string;
+  unit: string;
+  unitLong: string;
+  image: string;
+  id: number;
+}
+
+interface IngredientSearch {
+  id: number;
+  title: string;
+  image: string;
+  missedIngredientCount: number;
+  missedIngredients: MissedIngredients[];
+  usedIngredientCount: number;
+  usedIngredients: UsedIngredients[];
+}
+
+const IngredientSearchDefault: IngredientSearch = {
+  id: 0,
+  title: "",
+  image: "",
+  missedIngredientCount: 0,
+  missedIngredients: [{
+    amount: 0,
+    name: "",
+    unit: "",
+    unitLong:"",
+    image: "",
+    id: 0,
+  }],
+  usedIngredientCount: 0,
+  usedIngredients: [{
+    amount: 0,
+    name: "",
+    unit: "",
+    unitLong:"",
+    image: "",
+    id: 0,
+  }],
+}
 
-export default Recipe;
+export {Recipe, IngredientSearch, IngredientSearchDefault}
```

Comment:
we need to fix the linter -- this line shouldnt pass (spacing, semicolon missing)

Reply:
Will add spacing and punctuation